packages/react-core/src/components/Card/CardHeader.tsx

```diff
@@ -142,7 +144,11 @@ export const CardHeader: React.FunctionComponent<CardHeaderProps> = ({
           isDisabled: isCardDisabled,
           onChange: selectableActions.onChange,
           isChecked: selectableActions.isChecked ?? isSelected,
-          ...selectableActions.selectableActionProps
+          ...selectableActions.selectableActionProps,
+          style: {
+            ...selectableActions.selectableActionProps?.style,
+            ...(selectableActions?.isHidden && { visibility: 'hidden' })
+          }
```

Comment:
Rather than setting the inline style, we should add the `pf-v6-screen-reader` class to this input. Right now with the visibility hidden VO can't traverse to the input.

Reply:
@kmcfaul For the screen reader class, it'll have to go on the `input` element itself rather than the wrapper div